doc/how_to_guides/sbom_generation.md

```diff
@@ -0,0 +1,51 @@
+# How do I use CVE Binary Tool to generate a SBOM?
+
+The cve-bin-tool can be used to generate a software bill of materials (SBOM) file, which is a file that contains a list of all compoents detected by the scan in a standardized format.
+
+## SBOM support
+
+The cve-bin-tool generates SBOMs in the following formats
+
+| SBOM Type | Format   | Filename extension |
+| --------- | -------- | ------------------ |
+| SPDX      | TagValue | .spdx              |
+| SPDX      | JSON     | .spdx.json         |
+| SPDX      | YAML     | .spdx.yaml         |
+| SPDX      | YAML     | .spdx.yml          |
+| CycloneDX | JSON     | .json              |
+
+Details of the formats for each of the supported SBOM formats are available for [SPDX](https://spdx.dev/) and [CycloneDX](https://cyclonedx.org/)
+
+## Usage
+
+To generate a SBOM, run the tool as shown. See the examples below for details about optional arguments and default values used.
+
+```
+$ cve-bin-tool --sbom-type <sbom type> --sbom-format <sbom format> --sbom-output <sbom filename>
+```
+
+## Examples
+
+Generate a SPDX SBOM in TagValue format with the name sbom.spdx
+
+```
+cve-bin-tool --sbom-type spdx --sbom-format tagvalue --sbom-output sbom.spdx
```

Comment:
Looks like you'll need to add "tagvalue" to [allow.txt](https://github.com/intel/cve-bin-tool/blob/main/.github/actions/spelling/allow.txt) .  I guess only the capitalized version was added, since it wasn't caught above.